src/pages/call/calling.ts

```diff
@@ -0,0 +1,25 @@
+import { Component } from '@angular/core';
+import { NavController, NavParams } from 'ionic-angular';
+import { UserData } from '../../providers/user-data';
+
+@Component({
+  selector: 'page-calling',
+  templateUrl: 'calling.html',
+})
+export class CallingPage {
+    phone_number: string = '';
+
+    constructor(public navCtrl: NavController,
+                public navParams: NavParams,
+                public userData: UserData) {
+       this.phone_number = this.navParams.get('phone_number');
```

Comment:
@adrianq I don't know, it seems more straightforward (and shorter) the way I did it, and it still can do validations (pressing the "CALL" button calls the `call()` function in `call.ts`, where we could do it). Is it because a form would be more idiomatic? I'm happy to change it.

As a side note, your button icons and the placement of the bottom button is nicer in that page, so I'll use it as a reference to change `call.html`, thanks!